packages/dart/noports_core/lib/src/srv/srv_impl.dart

```diff
@@ -418,12 +420,19 @@ class SrvImplDart implements Srv<SocketConnector> {
       // bound to a port. Looking for specific output when the rv is ready to
       // do its job seems to be the only way to do this.
       if (detached) {
-        stderr.writeln(Srv.startedString);
+        try {
+          stderr.writeln(Srv.startedString);
+        } catch (e, st) {
+          logger.severe('Failed to write ${Srv.startedString}'
+              ' to stderr: ${e.toString()} ;'
+              ' stackTrace follows:\n'
+              '$st');
+        }
```

Comment:
If the child process _does_ encounter an exception from stderr.writeln, then catch it, log it, but continue to execute